plugins/modem-manager/fu-mm-utils.c

```diff
@@ -151,3 +151,40 @@ fu_mm_utils_get_port_info (const gchar	 *path,
 
 	return fu_mm_utils_get_udev_port_info (dev, out_device_bus, out_device_sysfs_path, out_port_usb_ifnum, error);
 }
+
+gboolean
+fu_mm_utils_find_device_file (const gchar	*device_sysfs_path,
```

Comment:
this function can it be moved into fu-udev-device?

Reply:
I can move it, but is it okay if it does not use FuUdevDevice type? I can only use FuDevice as a first parameter instead of passing physical id

Reply:
if I am adding new functions to libfwupdplugin, should I use `Since: 1.6.3`?

Reply:
left it in fu-mm-utils.c for now